lmdeploy/turbomind/supported_models.py

```diff
@@ -37,7 +37,8 @@
     MGMLlamaForCausalLM='llama',
     MiniGeminiLlamaForCausalLM='llama',
     # chatglm2/3, glm4
-    ChatGLMModel='glm4')
+    ChatGLMModel='glm4',
+    ChatGLMForConditionalGeneration='glm4')
```

Comment:
Can you share the huggingface repo id whose arch is defines as ChatGLMForConditionalGeneration?

Reply:
Please check if we need to update the unit tests

Reply:
It is https://huggingface.co/THUDM/glm-4-9b-chat/blob/main/config.json#L9
After quantization (loading & saving) the model architecture is converted to `ChatGLMForConditionalGeneration`

Reply:
Why change it to `ChatGLMForConditionalGeneration`

Reply:
I did not change it. `AutoModel.from_pretrained` automatically forms it.